torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -40,7 +40,7 @@ def resize_image(
 ) -> torch.Tensor:
     new_height, new_width = size
     num_channels, old_height, old_width = image.shape[-3:]
-    batch_shape = image.shape[:-3]
+    batch_shape = image.shape[:-3]  # In some places you use `image` to denote batches and in others `image_batch`. Should we align the names?
```

Comment:
The difference is that `image` is batch size agnostic whereas `image_batch` requires at least one batch dimension. Note that `batch_shape` here can very well be `()` if it is a single image without batch dimensions.